guides/vectorizing_guide.ipynb

```diff
@@ -0,0 +1,3643 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    dataset.vectorize(feature_vector=True)
</pre><p>FYI a <code>feature_vector</code> does not mean this when it comes to deep neural nets. <code>feature_vector</code>  should be called <code>numeric_vector</code>  IMO. Thoughts? See here: https://www.tensorflow.org/hub/common_signatures/images#:~:text=An%20image%20feature%20vector%20is,learned%20by%20the%20publisher%20model.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/RelevanceAI/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>